TombEngine/Scripting/Internal/TEN/Flow/Settings/Settings.cpp

```diff
@@ -196,6 +196,7 @@ namespace TEN::Scripting
 			"skinning", &GraphicsSettings::Skinning);
 	}
 
+	/* @fieldtype HairSettings[] */
```

Comment:
Can you elaborate on this change? What does it achieve?

Reply:
Since these are nested classes which should technically be split into their own modules, to avoid removing them from the main Settings module, I needed to add a custom property for the post processor to understand that this field is an array of settings.